javascript/schema/src/workflow/verbs.ts

```diff
@@ -677,6 +677,11 @@ export interface SampleArgs {
 	 * The randomization seed to use for sampling to ensure stable sampling.
 	 */
 	seed?: number
+
+	/**
+	 * Whether to preserve and emit the non-sampled records via the 'unsampled' output port.
+	 */
+	emitUnsampled?: boolean
```

Comment:
This serves two purposes: 

* It gets us *just the rows* that were not sampled, which may be useful for training/test dataset splitting.
* It allows us to have a core verb with multiple output tables for integration tests.

Reply:
Ok, on the first point I misunderstood the naming to mean the unsampled table, not the remainder rows. It might help to reword that slightly. Secondary suggestion though: is this actually needed as a param, or can the remainder just always be emitted as a secondary table?

Reply:
I like remainder, I think that's more descriptive. 

We _could_ always emit it, but it's more resource-intensive to do so